### PR TITLE
[feat] Creates the skeleton for the team_statistics_spec.rb file

### PR DESCRIPTION
**Pull request by MiTOBrien**

### CODE REVIEW

File: spec/team_statistics_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    team_path = './data/teams_dummy.csv'
+    game_teams_path = './data/game_teams_dummy.csv'
+
+    locations = {
+      games: game_path,
+      teams: team_path,
+      game_teams: game_teams_path
+    }
+    @stat_tracker = StatTracker.from_csv(locations)
+    @team_statistics = TeamStatistics.new(@stat_tracker.teams, @stat_tracker)
```
Comment (sethverrill) [inline spec/team_statistics_spec.rb]: Do we only need more than teams for these stats? looks good otherwise.

**Comment by ossar3** [comment]
looks good to me

File: spec/team_statistics_spec.rb (hunk 1 above)
Comment (MiTOBrien) [reply spec/team_statistics_spec.rb]: Good call.  It references seasonal data so I added games as well so we can parse season data.

**Comment by sethverrill** [comment]
Nice!

**Pull request merged**